package.json

```diff
@@ -94,7 +94,8 @@
     "socket.io-client": "2.3.1",
     "tasklist": "^3.1.1",
     "uuid": "^3.0.1",
-    "vuejs-datepicker": "^1.6.2"
+    "vuejs-datepicker": "^1.6.2",
+    "zustand": "^4.4.1"
```

Comment:
looks good so far! is there a way we can move this to devDependencies or does it need to be here? also, no issues from not being on TS5?

regardless, we should hold off on merging until after dual output release just to make sure the app is in a stable state if we need to add some dual output fixes in a bundle

Reply:
There were some issues with types, but I found a workaround. Yeah, Zustand should be in dev deps.